spec/features/users_feature_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+describe 'users' do
+
+  context 'visiting root path and havent logged in' do
+
+    scenario 'is promted a sign in and sign up link' do
+      visit '/'
+      expect(page).to have_content 'Sign in'
+      expect(page).to have_content 'Sign up'
+    end
+
+    scenario 'clicks on sign up redirects to sign up path and promts a sign up form ' do
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)